products/firefox_iOS/adding_new_iOSlocale.md

```diff
@@ -0,0 +1,43 @@
+# Adding a new locale to Firefox for iOS
+
+## Prerequisites
+
+### iOS language support
+
+iOS currently supports [two tiers of locales](https://people-mozilla.org/~sarentz/ios-locales.html), and a locale must be part of that list in order to ship on Firefox for iOS. In fact, there is no locale switcher on Firefox for iOS (vs Firefox on Android, which does have one) - so if a locale is supported totally depends on what languages the system supports itself.
+
+Note that for Tier 2 locales, some menu items in Firefox for iOS can not be translated since they are part of the OS itself. Strings will unfortunately appear in English on the final UI in these cases.
+
+To tell if a locale is supported or not, it is also possible simply to take a look on an actual device, under the Language Settings of the most recent iOS version available. Tier one locales are the ones listed under “iPod Languages”, and Tier 2 locales appear under “Other Languages”. If a locale appears under none of these two lists, then it should be safe to assume the language is not supported by iOS (unless it is a right-to-left language, see later below) - and therefore we can not ship it as part of Firefox for iOS.
```

Comment:
Need to check on devices how "iPod Languages" look on iPhones (we have three in the room).

The note about RTL languages it's not true: they're supported by the OS, you make it sound like they're not, while we're not supporting them properly in the app.

we can not ship -> it's not possible to ship